faq.html

```diff
@@ -142,39 +166,40 @@ <h3 class="question">How do I contribute to RDS?</h3>
 
               <!-- prettier-ignore -->
               <ul class="faq__list">
-              <li><a href="https://github.com/Real-Dev-Squad/website-welcome" target="_blank"
+              <li><a tabindex="-1" href="https://github.com/Real-Dev-Squad/website-welcome" target="_blank"
```

Comment:
Why are we using tab-index -1?

Reply:
To hide the the links when accordion is closed